react/components/DrawerMode.tsx

```diff
@@ -12,13 +12,15 @@ interface Props {
   maxDrawerWidth: number | string
   drawerSlideDirection: SlideDirectionType
   quantityDisplay: 'always' | 'never' | 'not-empty'
+  showTotalItemsQty: boolean
```

Comment:
Apart from the comment about this becoming an `enum`, this prop is being drilled pretty deep... it might be a better idea to add it to the `MinicartContext`.

Reply:
same thing with `quantityDisplay`